chain/chain/src/flat_storage_creator.rs

```diff
@@ -45,30 +59,104 @@ impl FlatStorageShardCreator {
             shard_id,
             start_height,
             runtime_adapter,
-            fetched_state_parts: None,
+            remaining_state_parts: None,
             fetched_parts_sender,
             fetched_parts_receiver,
+            visited_trie_items: 0,
         }
     }
 
+    #[allow(unused)]
+    fn state_part_hex_prefix(path_begin: &[u8]) -> String {
```

Comment:
Could you add a comment for this function? It's a bit hard to figure out what it does.